Sources/ProjectDescription/RunActionOptions.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// Options for the `RunAction` action
+public struct RunActionOptions: Equatable, Codable {
+    /// The path of the
+    /// [StoreKit configuration file](https://developer.apple.com/documentation/xcode/setting_up_storekit_testing_in_xcode#3625700).
+    public let storeKitConfigurationPath: Path?
+
+    /// Creates an `RunActionOptions` instance
+    ///
+    /// - Parameters:
+    ///     - storeKitConfigurationPath: The path of the
+    ///     [StoreKit configuration file](https://developer.apple.com/documentation/xcode/setting_up_storekit_testing_in_xcode#3625700).
+    ///     Please note that this file is automatically added to the Project/Workpace. You should not add it manually.
+    ///     The default value is `nil`, which results in no configuration defined for the scheme
+    public init(
```

Comment:
I'd recommend keeping this one internal, and rather exposing a static method:

```swift
RunAction(options: .options(storeKitConfigurationPath: "xxx"))
```

It reads better than `.init`

Reply:
Sure. Out of curiosity, why do you prefer this way?